### PR TITLE
Updated OpenAPI contract

### PR DESCRIPTION
**Pull request by yashodgayashan**

### CODE REVIEW

File: BackendNew/resources/openapi.yaml [hunk 1]
```diff
@@ -277,7 +277,7 @@ paths:
                     type: string
         "404":
           description: students not found
-  /student/{student-id}/feedback:
+  /student/{student-id}/feedbacks:
```
Comment (hasinichathu) [inline BackendNew/resources/openapi.yaml:280]: simple spelling mistake. It should be "feedback"

File: BackendNew/resources/openapi.yaml [hunk 2]
```diff
@@ -1212,6 +1212,37 @@ paths:
...
+    BasicStudent:
+      type: object
+      properties:
+        id:
+          type: string
+          description: "The index number of the student"
+        batch:
+          type: string
+          description: "The intern batch of the student"
+        fullName:
```
Comment (hasinichathu) [inline BackendNew/resources/openapi.yaml:1248]: Did we agreed upon to add both full name and name with initials by admin at initial phase?

Comment (yashodgayashan) [reply BackendNew/resources/openapi.yaml:1248]: Will make it as Name with initials

File: BackendNew/resources/openapi.yaml (hunk 1 above)
Comment (yashodgayashan) [reply BackendNew/resources/openapi.yaml:280]: It should be since it is a collection resource

Comment (hasinichathu) [reply BackendNew/resources/openapi.yaml:280]: no word like "feedbacks"

Comment (yashodgayashan) [reply BackendNew/resources/openapi.yaml:280]: I will change it

**Pull request merged**